decidim-meetings/lib/decidim/meetings/engine.rb

```diff
@@ -27,6 +27,7 @@ class Engine < ::Rails::Engine
           resource :meeting_widget, only: :show, path: "embed"
         end
         root to: "meetings#index"
+        resource :calendar, only: [:show], format: :text
```

Comment:
Why the route definition is different in the two engines?

Reply:
Because I reused the `meetings_controller` in the `directory` engine, but I can fix it if you think it's better. Which solution would you prefer?

Reply:
It's OK I guess, I just was wondering why one had the default format and the other didn't.